### ISSUE TITLE
dep-graph generation fails on latest version of nx

### ISSUE THREAD

**Issue by cladera** [khalilou88/jnxplus#41]
## Reproduce steps
1. Create two boot-maven libraries, lib1 and lib2
2. Add lib1 to lib2's dependencies
3. Run `nx dep-graph`

## Output

```
❯ NX_VERBOSE_LOGGING=true nx dep-graph
TypeError: (0 , app_root_1.fileExists) is not a function
    at Object.processProjectGraph (/Users/cgalancladera/foo/node_modules/@jnxplus/nx-boot-maven/src/dep-graph/lookup-deps.js:14:39)
    at /Users/cgalancladera/foo/node_modules/nx/src/project-graph/build-project-graph.js:212:38
    at Generator.next (<anonymous>)
    at /Users/cgalancladera/foo/node_modules/tslib/tslib.js:118:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/cgalancladera/foo/node_modules/tslib/tslib.js:114:16)
    at updateProjectGraphWithPlugins (/Users/cgalancladera/foo/node_modules/nx/src/project-graph/build-project-graph.js:207:20)
    at /Users/cgalancladera/foo/node_modules/nx/src/project-graph/build-project-graph.js:89:25
    at Generator.next (<anonymous>)
    at fulfilled (/Users/cgalancladera/foo/node_modules/tslib/tslib.js:115:62)
Failed to process the project graph with "index.js". This will error in the future!
TypeError: (0 , app_root_1.fileExists) is not a function
    at Object.processProjectGraph (/Users/cgalancladera/foo/node_modules/@jnxplus/nx-boot-maven/src/dep-graph/lookup-deps.js:14:39)
    at /Users/cgalancladera/foo/node_modules/nx/src/project-graph/build-project-graph.js:212:38
    at Generator.next (<anonymous>)
    at /Users/cgalancladera/foo/node_modules/tslib/tslib.js:118:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/cgalancladera/foo/node_modules/tslib/tslib.js:114:16)
    at updateProjectGraphWithPlugins (/Users/cgalancladera/foo/node_modules/nx/src/project-graph/build-project-graph.js:207:20)
    at /Users/cgalancladera/foo/node_modules/nx/src/project-graph/build-project-graph.js:89:25
    at Generator.next (<anonymous>)
    at fulfilled (/Users/cgalancladera/foo/node_modules/tslib/tslib.js:115:62)
Failed to process the project graph with "index.js". This will error in the future!
```

**Comment by khalilou88** [comment]
@cladera Hi,
I published the version 2.1.1 with your fix. 
I also uses patch releases for deps to prevent this kind of issues.

**Comment by khalilou88** [comment]
I ended updating NX to the version 14.2.4. Please use the version 2.2.0
